thunderfish/csvmaker.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+from .dataloader import load_data
+from .consistentfishes import consistent_fishes
+from .harmonicgroups import harmonic_groups
+from .powerspectrum import multi_resolution_psd
+
+
+def extract_main_freqs_and_db(data, samplerate):
```

Comment:
I agree, that I don't need it in thunderfish. But I do need it in the independent csvmaker.py module. So what do we suggest doing??